### PR TITLE
Implemented polylabel definition

### PR DESCRIPTION
**Pull request by DenisCarriere**
Definitions is tested

@mourner Please review this Typescript definition

### CODE REVIEW

File: polylabel/polylabel.d.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     * @function
+     * @param {Array<number>} polygon - Given polygon coordinates in GeoJSON-like format
+     * @param {number} precision - Precision (1.0 by default)
+     * @param {boolean} debug - Debugging for Console 
+     * @return {Array<number>}
+     * @example
+     * var p = polylabel(polygon, 1.0);
+     */
+    function polylabel (polygon: number[][][], precision?: number, debug?: boolean): number[];
+    export default polylabel;
```
Comment (RyanCavanaugh) [inline polylabel/polylabel.d.ts]: This should be `export = polylabel` based on how the code is written. It does not produce a default export.

**Comment by DenisCarriere** [comment]
@RyanCavanaugh Do you want me to create a new PR? Or are the 3 commits fine?

**Comment by RyanCavanaugh** [comment]
One PR's fine

**Comment by DenisCarriere** [comment]
@RyanCavanaugh Thanks! DT is truly awesome! All repos should be included in DT. I'll do my best to send over PRs.
Cheers,

**Pull request merged**